lib/modules/featurePenaltyBox.js

```diff
@@ -0,0 +1,176 @@
+/* @flow */
+
+import { $ } from '../vendor';
+import * as Options from '../core/options';
+import * as Modules from '../core/modules';
+import { Module } from '../core/module';
+import { i18n } from '../environment';
+import { mapScalarToObject } from '../utils';
+import * as Notifications from './notifications';
+import * as SettingsNavigation from './settingsNavigation';
+
+export const module: Module<*> = new Module('resPenaltyBox');
+export const MIN_PENALTY = 1;
+export const MAX_PENALTY = 100;
+
+module.moduleName = 'resPenaltyBoxName';
+module.category = 'coreCategory';
+module.description = 'resPenaltyBoxDesc';
+module.options = {
+	delayFeatures: {
+		type: 'boolean',
+		value: true,
+		description: 'resPenaltyBoxDelayFeaturesDesc',
+	},
+	suspendFeatures: {
+		type: 'boolean',
+		value: false,
+		description: 'resPenaltyBoxSuspendFeaturesDesc',
+	},
+	features: {
+		description: 'resPenaltyBoxFeaturesDesc',
+		type: 'table',
+		advanced: true,
+		addRowText: 'manually register feature',
+		fields: [
+			{
+				name: 'moduleID',
```

Comment:
How about [`id`](https://github.com/honestbleeps/Reddit-Enhancement-Suite/pull/3587/commits/0f2bc0eeb4c49c64337fc5c4082a73e803115ae1) with a fallback to `name`?